From 56023dc4c5c42bef5b31382b64d2a3d701a57fda Mon Sep 17 00:00:00 2001
From: Oleg Kolosov <bazurbat@gmail.com>
Date: Mon, 6 Feb 2017 11:24:07 +0100
Subject: [PATCH] pass all LDFLAGS of the found CommonAPI to the build

---
 AudioManagerUtilities/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/AudioManagerUtilities/CMakeLists.txt b/AudioManagerUtilities/CMakeLists.txt
index 98abc69..a50caea 100644
--- a/AudioManagerUtilities/CMakeLists.txt
+++ b/AudioManagerUtilities/CMakeLists.txt
@@ -77,7 +77,7 @@ IF (WITH_CAPI_WRAPPER)
 
     	set(AUDIO_MANAGER_UTILITIES_LIBS
         	${AUDIO_MANAGER_UTILITIES_LIBS}
-       		${COMMONAPI_LIBRARIES}
+		${COMMONAPI_LDFLAGS}
         	${COMMONAPI_DBUS_LIBRARIES}
 		CACHE INTERNAL "AUDIO_MANAGER_UTILITIES_LIBS libs" FORCE)
 
-- 
2.10.2

